pennylane_orquestra/forest_device.py

```diff
@@ -15,6 +15,7 @@
 The Forest device class for PennyLane-Orquestra.
 """
 from pennylane_orquestra.orquestra_device import OrquestraDevice
+import warnings
```

Comment:
How come this change was made?

Reply:
It was a missed import, added a test case too using `pytest.mark.parametrize` to cover this:
```python
non_analytic_backends = [
    ("orquestra.qiskit", "qasm_simulator"),
    ("orquestra.forest", "3q-noisy-qvm"),
]
```